proto/adaptors/uniswap/uniswap_v3.proto

```diff
@@ -29,6 +29,12 @@ message UniswapV3Adaptor {
         TakeFromPosition take_from_position = 7;
         // Represents function `collectFees(uint256 positionId, uint128 amount0, uint128 amount1)`
         CollectFees collect_fees = 8;
+        // Represents function `purgeAllZeroLiquidityPositions(ERC20 token0, ERC20 token1)`
+        PurgeAllZeroLiquidityPositions purge_all_zero_liquidity_positions = 9;
```

Comment:
I think the 2.x line failed to do this, but we should probably re-run the doc generation.